src/js/lib/monster.js

```diff
@@ -555,10 +555,14 @@ define(function(require) {
 			self.kazooSdk = $.getKazooSdk({
 				apiRoot: monster.config.api.default,
 				onRequestStart: function(request, requestOptions) {
-					monster.pub('monster.requestStart');
+					if (!_.get(requestOptions, 'preventRequestEvents', false)) {
```

Comment:
So, this works but it completely disables the requestStart/requestEnd events, which could be used for something else.

In fact, it is used by `insertTemplate()` to determine if a request is currently running.

A better way to do this would be to keep the event publishing but disable the spinner.